httpserver/service_node.cpp

```diff
@@ -2,8 +2,10 @@
 
 #include "Database.hpp"
 #include "Item.hpp"
+#include "custom_formatters.h"
```

Comment:
Would we ever want to include `custom_formatters.h` and not `loki_logger.h`?

Reply:
I've changed to overload the `<<` operator for `sn_record_t` directly in the class instead.